scripts/abundance-dist.py

```diff
@@ -87,7 +87,9 @@ def main():
     print('HT sizes:', hashsizes, file=sys.stderr)
     print('outputting to', args.output_histogram_filename, file=sys.stderr)
 
-    if os.path.exists(args.output_histogram_filename):
+    if args.output_histogram_filename in ('-', '/dev/stdin'):
+        pass
+    elif os.path.exists(args.output_histogram_filename):
```

Comment:
Your _output_ filename shouldn't be `/dev/stdin`. I think you meant `/dev/stdout`